internal/ingress/metric/collectors/controller.go

```diff
@@ -305,6 +305,7 @@ func (cm *Controller) SetSSLExpireTime(servers []*ingress.Server) {
 		}
 		labels["host"] = s.Hostname
 		labels["secret_name"] = s.SSLCert.Name
+		labels["secret_namespace"] = s.SSLCert.Namespace
```

Comment:
I think this should be namespace and not secret_namespace, like for other metrics (including ssl_info).

Reply:
Yeah, sure, changed it!